extensions/binding/org.eclipse.smarthome.binding.ble.bluegiga/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BlueGiga BLE Bridge Binding Fragment
+Bundle-SymbolicName: org.eclipse.smarthome.binding.ble.bluegiga;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/com.zsmartsystems.bluetooth.bluegiga-1.0.0-SNAPSHOT.jar
+Fragment-Host: org.eclipse.smarthome.binding.ble
+Import-Package: 
+ gnu.io;version="3.12.0.OH",
```

Comment:
Shouldn't we try to work with the Kura team and use  + improve that EPL licensed serial driver?
https://github.com/eclipse/kura/tree/develop/target-platform/org.eclipse.soda.dk.comm-parent/org.eclipse.soda.dk.comm/src/main/java
It does not use gnu.io but the javax.comm interface.
But if it is working it does not matter.
Perhaps we can create an abstraction layer, so the real "backend driver" does not care.
But IMHO it would be a big step if we could use an EPL licensed serial driver.